src/ext/java/org/opentripplanner/ext/transmodelapi/TransmodelGraphQLSchema.java

```diff
@@ -943,7 +943,7 @@ private GraphQLSchema create() {
               Collection<VehicleRentalPlace> all = new ArrayList<>(GqlUtil
                   .getRoutingService(environment)
                   .getVehicleRentalStationService()
-                  .getVehicleRentalStations());
+                  .getVehicleRentalPlaces());
```

Comment:
I think we would like to keep the same behaviour as legacy GraphQL endpoint has here

```suggestion
                  .getVehicleRentalStations());
```